app/javascript/src/App.jsx

```diff
@@ -1,5 +1,7 @@
 import React from "react";
 
+import dayjs from "src/lib/dayjs"; // eslint-disable-line
```

Comment:
- Add alias for `lib` path.
- No need to import with a name. Here the idea is to initialise the dayjs singleton. Our initialization logic extends the singleton instance. We need to import under a name only when we are utilising the imported name within the component.
```suggestion
import "lib/dayjs"; // eslint-disable-line
```